packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -19,6 +19,7 @@ import TokenBalance from '../../Tokens/TokensList/TokenBalance';
 import withAccount from '../../utils/withAccount.js';
 import withBalance, { withEthBalance } from '../../utils/withBalance';
 import withTokens from '../../utils/withTokens';
+import createDecorator from 'final-form-calculate';
```

Comment:
This probably is added by your IDE. This import should go in the top block (absolute imports), and ordered by import name.